packages/inventory-general-info/src/__snapshots__/GeneralInformation.test.js.snap

```diff
@@ -1136,7 +1184,7 @@ exports[`GeneralInformation should render correctly 1`] = `
               class=""
               data-pf-content="true"
             >
-              13 Aug 2019
+              12 Aug 2019
```

Comment:
This should actually be `13 Aug 2019` because of timezones and such.

Reply:
oh, I missed that one, thanks!